### ISSUE TITLE
Google Play Store listing

### ISSUE THREAD

**Issue by FormularSumo** [FormularSumo/Star-Wars-Force-Collection-Remake#40]
On 6 April I made a Google Play developer's account and submitted this game. I got internal testing working on the same day but public testing (closed/open) had to wait until the game's been approved by Google.
https://play.google.com/apps/internaltest/4698311765937061601 (internal testing page)
![image](https://user-images.githubusercontent.com/36278229/115989150-d7506a80-a5b4-11eb-8f14-aa8338fbea41.png)
![image](https://user-images.githubusercontent.com/36278229/115989160-e0d9d280-a5b4-11eb-885a-a6f18fb70d3a.png)

The first few times I submitted it was rejected for these reasons. I don't actually have a promo video for my app so I assume whoever was reviewing just ticked all the boxes.
[This](https://docs.google.com/document/d/1byO6tONbLSp6fiS2z8zM9zzbytuskc25wgCKwmUS16Q/edit) is my private policy. I improved it using https://support.google.com/googleplay/android-developer/answer/9893335 and https://tos.ea.com/legalapp/WEBPRIVACY/US/en/PC and in the most recent review my app wasn't rejected due to the private policy so I think that's been fixed.

The 2nd issue was with using copyrighted content in my app listing, so I've removed it all. First by removing 'Star Wars' from the name/logo/description, then by removing screenshots of my game with Star Wars characters in. In the most recent review mentioned above my app was only rejected for these reasons 
![image](https://user-images.githubusercontent.com/36278229/115989291-9573f400-a5b5-11eb-8617-c7ee00625e67.png) so I renamed the game from 'Force Collection' to 'Galaxy Collection'.

**Comment by FormularSumo** [comment]
![galaxy_collection-playstore](https://user-images.githubusercontent.com/36278229/115989787-f69cc700-a5b7-11eb-9892-70ec423c997a.png)
New logo

**Comment by FormularSumo** [comment]
Game got approved yesterday, so changing the name and logo worked. I've disabled internal testing and alpha testing track as I don't really need them and I pushed the open testing release to production channel. So my game is now listen on the play store, and can be discovered and installed by anyone! The open beta channel is still open and can't be signed up the store listing page.

![Screenshot_20210428-134852.png](https://user-images.githubusercontent.com/36278229/116406486-a1b3b780-a828-11eb-8ddc-ef9afc901f0e.png)
